### PR TITLE
Experiences from elasticon

### PR DESCRIPTION
**Pull request by jazminschroeder**

### CODE REVIEW

File: _posts/2015-03-22-experiences-from-elasticon.markdown [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title:  "Experiences from Elastic{ON}"
+date:   2015-03-22 20:00:00"
+categories: conferences elasticon
+comments: false
```
Comment (ppeble) [inline _posts/2015-03-22-experiences-from-elasticon.markdown]: This won't hurt but FYI this is ignored from now on. We don't allow comments anymore.

File: _posts/2015-03-22-experiences-from-elasticon.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+By far this was the most popular track due to presentations given by companies like Cisco, USGS, Mozilla, Verizon, Yale, Github, Wikimedia, Yelp, Fico, Netflix and Zendesk among others. It was literally standing room only or no room at all. A couple of presentations I was able to squeeze in:  
+
+1. **”Tectonic Shifts, Tweets, & Tactical Data Analysis”** [presented by Michelle Guy from USGS](https://www.elastic.co/elasticon/2015/sf/tectonic-shifts-tweets-and-tactical-data-analysis/?view=1)
+
+>> Apparently, instrument based earthquake monitoring systems may take up to 10 minutes to broadcast information regarding the epicenter of an earthquake. USGS has a project currently in Beta which analyzes tweet trends during seismic activity which has reduced the feedback loop to less than 2 minutes. The tweet data is geocoded and put into Elasticsearch, a pretty map in a Kibana dashboard does the rest. How cool is that?
+
+
+2. **”Scaling Elasticsearch for Production at Verizon: 500 Billion Documents & Counting”** [presented by Bhaskar Karambelkar from Verizon](https://www.elastic.co/elasticon/2015/sf/scaling-elasticsearch-for-production-at-verizon/?view=1)
```
Comment (ppeble) [inline _posts/2015-03-22-experiences-from-elasticon.markdown]: This doesn't actually seem to register as '2' on my screen. It thinks you are starting a new list.

File: _posts/2015-03-22-experiences-from-elasticon.markdown [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+>> Doesn't the title say it all? Amazing talk! What to do, and more importantly, what not to do from a company that is indexing over 10 billion entry logs daily, and with an impressive 500 billion plus documents to date. [Definitely check out the video](https://www.elastic.co/elasticon/2015/sf/scaling-elasticsearch-for-production-at-verizon/?view=1)
+
+#### Track 2 - Developer track(Inside look at what's coming up in Elastic)
```
Comment (ppeble) [inline _posts/2015-03-22-experiences-from-elasticon.markdown]: Nitpick but I think that there should be a space after 'track', between it and the parenthesis.

File: _posts/2015-03-22-experiences-from-elasticon.markdown [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Elastic engineers are really passionate about their products and were eager to share. This track was full of insights and lessons learned from the team. What’s coming up? Improvements to Elasticsearch 2.x, a lot of the things we don’t see. More graphs and better filters for Kibana as well as more reliability for Logstash. If you are looking for a technical flavor check out this track talks.
+
+#### Track 3 - Demo theater(Demos and more demos)
```
Comment (ppeble) [inline _posts/2015-03-22-experiences-from-elasticon.markdown]: Same spacing comment as above.

**Comment by ppeble** [comment]
Just a few formatting points, looks great!

**Comment by ppeble** [comment]
@jazminschroeder Would you like to close this, then? Is it too late to post it?

**Comment by jazminschroeder** [comment]
Yup.. closing it

**Pull request closed without merging**